File: python/bulls.py
startingLineup = [
    {
        "playerName": "Ron Harper",
        "jerseyNumber": 9,
        "college": "Miami University",
        "position": "Guard",
        "captain": False,
    },
    {
        "playerName": "Tony Kukoc",
        "jerseyNumber": 7,
        "college": "Croatia",
        "position": "Forward",
        "captain": False,
    },
    {
        "playerName": "Dennis Rodman",
        "jerseyNumber": 91,
        "college": "Southeastern Oklahoma State University",
        "position": "Forward",
        "captain": False,
    },
    {
        "playerName": "Scottie Pippen",
        "jerseyNumber": 33,
        "college": "Central Arkansas",
        "position": "Forward",
        "captain": False,
    },
    {
        "playerName": "Michael Jordan",
        "jerseyNumber": 23,
        "college": "North Carolina",
        "position": "Guard",
        "captain": True,
    },
]

for player in startingLineup:
    position = player["position"]
    jerseyNumber = player["jerseyNumber"]
    college = player["college"]
    playerName = player["playerName"]
    
    playerString = "At " + position + ", number " + str(jerseyNumber) + ", from " + college
    
    if player["captain"] == True:
        playerString += ", your captain"
    playerString += ", " + playerName.upper() + "!"
    
    print(playerString)

File: rust/src/bulls.rs
// import std library
use std::fmt;

// create a player struct
struct Player {
    name:String,
    number:u8,
    college:String,
    position:String,
    captain:bool
}

// implement player struct
impl Player {
    // create a new player
    fn new (name:&str, number:u8, college:&str, position:&str, captain:bool) -> Player {
        Player {
            name:name.to_string(),
            number:number,
            college:college.to_string(),
            position:position.to_string(),
            captain:captain
        }
    }
}

// allow player struct to display in console
impl fmt::Display for Player {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(
            f,
            "({}, {}, {}, {}, {})",
            self.name,
            self.number,
            self.college,
            self.position,
            self.captain
        )
    }
}

// run
pub fn run() {

    // create the 5 players
    let player1:Player = Player::new(
        "Ron Harper",
        9,
        "Miami University",
        "Guard",
        false
    );
    
    let player2:Player = Player::new(
        "Tony Kukoc",
        7,
        "Croatia",
        "Forward",
        false
    );

    let player3:Player = Player::new(
        "Dennis Rodman",
        91,
        "Southeastern Oklahoma State University",
        "Forward",
        false
    );

    let player4:Player = Player::new(
        "Scottie Pippen",
        33,
        "Central Arkansas",
        "Forward",
        false
    );

    let player5:Player = Player::new(
        "Michael Jordan",
        23,
        "North Carolina",
        "Guard",
        true
    );

    // create an array to iterate through
    let lineup:[Player; 5] = [player1, player2, player3, player4, player5];

    // iterate and print each Player
    for player in lineup.iter() {
        if !player.captain {
            println!(
                "Playing {}, number {}, from {}, {}!",
                player.position,
                player.number.to_string(),
                player.college,
                player.name.to_uppercase()
            );
        } else {
            println!(
                "Playing {}, number {}, from {}, your captain, {}!",
                player.position,
                player.number.to_string(),
                player.college,
                player.name.to_uppercase()
            );
        }
    }
}
